pkgs/development/libraries/portaudio/default.nix

```diff
@@ -12,30 +13,26 @@ stdenv.mkDerivation rec {
     ++ stdenv.lib.optional (!stdenv.isDarwin) alsaLib;
 
   configureFlags = stdenv.lib.optionals stdenv.isDarwin
-    [ "--build=x86_64" "--without-oss" "--enable-static" "--enable-shared" ];
+    [ "--disable-mac-universal" ];
```

Comment:
That's what I mean. Does it need to be conditional at all? Wouldn't a linux build just ignore the flag?